spec/models/claim/pii_scrubber_spec.rb

```diff
@@ -0,0 +1,102 @@
+require "rails_helper"
+
+RSpec.describe Claim::PiiScrubber, type: :model do
+  let(:over_two_months_ago) { 2.months.ago - 1.day }
+
+  it "does not delete details from a submitted claim" do
+    freeze_time do
+      claim = create(:claim, :submitted, updated_at: over_two_months_ago)
+
+      Claim::PiiScrubber.new.scrub_completed_claims
+      cleaned_claim = Claim.find(claim.id)
+
+      expect(cleaned_claim.attributes).to eq(claim.attributes)
```

Comment:
We could use the Rspec change matcher to rewrite these "not delete" specs like this:

```ruby
  expect { Claim::PiiScrubber.new.scrub_completed_claims }.not_to change { claim.reload.attributes }
```